### ISSUE TITLE
should support to keep some number of recently published of artifacts

### ISSUE THREAD

**Issue by hoatle** [c-hive/gha-remove-artifacts#26]
thank for your helpful action, can you add support for keeping a configurable number of recently published artifacts?

For example:

```
    steps:
    - name: Remove old artifacts
      uses: c-hive/gha-remove-artifacts@v1
      with:
        age: '1 month'
        keep-recent: 7 # keep the number of recently published artifacts 
        skip-tags: true
```

so that even if those 7 recently published artifacts are older than 1 month, the action will not delete them.

**Comment by thisismydesign** [comment]
Thanks for the feedback. Let's name the config `skip-recent`, otherwise one might understand them as competing options (i.e. if you have 10 artifacts within 1 month with your example config it's not clear to me if it should keep 7 or 10).

@gomorizsolt feel free to take a crack at this.

**Comment by gomorizsolt** [comment]
Added this option in https://github.com/c-hive/gha-remove-artifacts/releases/tag/v1.2.0.

**Comment by thisismydesign** [comment]
Released in [v1.2.0](https://github.com/c-hive/gha-remove-artifacts/releases/tag/v1.2.0), implemented in https://github.com/c-hive/gha-remove-artifacts/pull/27
